src/config/dynamic-theme-fixes.config

```diff
@@ -3684,6 +3684,9 @@ div[role="menuitem"] svg
 div[role="dialog"][style^="width: 350px;"] div[role="button"][tabindex="0"]
 div[role="dialog"] ~ div[role="menu"] > div[role="menuitem"] > div > div:not([style*="background-image"])
 img[src$="type/audio/mp3"]
+div[role="listitem"] > div > div > div > svg
+div[role="menuitem"] > div > div > div
```

Comment:
Only this one concerns me as every "right-click menu" has such class and will definitely have more than 3 div's inside of it. Do you have an example wherefor this is used?

Reply:
![imagen](https://user-images.githubusercontent.com/71190696/126425579-b8952030-878b-49b0-a9b5-73498e2a5e90.png)
I don't exactly understand what you mean, but Line 3688 is for the icons from the "add shortcut to drive" menu that appears once you select the option from the main right-click menu. I tried, but I don't think it is within my ability to narrow this one down further.

On the other hand, would changing Line 3687 to `div[role="listitem"] > div > div > div > svg[fill="#000000"] > path` be of any help in narrowing that one down?

Reply:
The 2 other lines don't concern me for now, but ` svg[fill="#000000"] > path` would be helpful.

Welp, we actually encounter such `invert` problem from the drive.google.com/file viewer and with a slightly modified CSS the same trick will work
```CSS
div[role="menu"] > div > div[role="menuitem"] > div > div > div
div[role="menu"] > div > div[role="menuitem"][class*=" "] > div > :last-child
```